pkg/api/v1/nonstandard_types.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+type Sysctl struct {
```

Comment:
They are for alpha features and were previously located in helpers.go instead of in types.go. According to offline discussion, there's no reason not putting them in types.go, so I moved them to types.go. @lavalamp PTAL, thanks.